demos/nextAuth-demo/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
```

Comment:
Just wondering if this is the correct license or not? MIT would make sense to me, but I believe we need to have the same license as projects we reference.